test/JDBC/expected/BABEL-UNSUPPORTED.out

```diff
@@ -2781,28 +2781,12 @@ GO
 
 CREATE TABLE test_unsupported_geography(a GEOGRAPHY);
```

Comment:
This whole section can go, since it is supported now.

Reply:
+1, let's remove all these commands now since they are now supported.

Reply:
Resolved